rubicon_ml/client/feature.py

```diff
@@ -49,3 +55,8 @@ def importance(self):
     def created_at(self):
         """Get the feature's created_at."""
         return self._domain.created_at
+
+    @property
+    def parent(self):
+        """Get the metrics's parent client object."""
```

Comment:
```suggestion
        """Get the feature's parent client object."""
```

Reply:
also if it says "metrics's" with the extra s where you copied this from in metric.py can you fix that typo too?